### ISSUE TITLE
各位兄弟有人知道tensorRT的conv前如何设置不对称padding么？

### ISSUE THREAD

**Issue by Carnon** [wang-xinyu/tensorrtx#830]
conv->setPaddingNd(DimsHW{3, 3})  这个是这是设置上下左右都是3的填充，   我想设置上2，下4， 左2，右4的填充。类似tensorflow的    tf.keras.layers.ZeroPadding(padding=(2, 4,), (2, 4))的操作

**Comment by Carnon** [comment]
真尴尬 用conv->setPrePadding(DimsHW{2, 2})   和conv->setPostPadding(DimsHW{4, 4}) 就可以了  对英文不太熟悉，瞎试出来的
